src/lambda/crudHandler.ts

```diff
@@ -0,0 +1,87 @@
+import { lambdaGetParameters } from './lambdaGetParameters'
+import { APIGatewayEvent } from 'aws-lambda'
+import { error } from '../error'
+import { getDefaultResponse, HttpNames } from '../http'
+
+export const lambdaCrudHandler = (event: APIGatewayEvent): object => {

```

Comment:
´object´ isn't a very useful type in here. I'd suggest letting TypeScript infer the return type of those functions (and disabling ´@typescript-eslint/explicit-function-return-type´ for the file).

Reply:
let's talk about it in the chatper

Reply:
so I just leave it the way it was @Viserion77 ?

Reply:
@Viserion77 and I agreed to leave it like that. We can type this later.